submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/SubmitterConfig.kt

```diff
@@ -202,16 +204,18 @@ class SubmitterConfig {
 
     @Configuration
     @Suppress("MagicNumber")
+    @EnableConfigurationProperties
     class ServiceConfig(
         private val service: PersistenceService,
         private val queryService: SubmissionMetaQueryService,
         private val userPrivilegesService: IUserPrivilegesService,
+        @Value("\${app.subBasePath:#{null}}") private val subBasePath: String?,
```

Comment:
Why are we using a value notation instead of adding it to the [corresponding object](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/ApplicationProperties.kt#L8)?